lighthouse-core/lib/i18n/README.md

```diff
@@ -202,6 +202,7 @@ CTC is a name that is distinct and identifies this as the Chrome translation for
   "name": {
     "message": "Message text, with optional placeholders.",
     "description": "Translator-aimed description of the message.",
+    "meaning": "Description given when a message is duplicated, in order to give context to the message.",
```

Comment:
isn't this just a copy of `description` for us? seems like this doc is a good place to diverge from the chrome docs if we're doing something different :)

Reply:
Yup. Just having a `meaning` in our spec is the divergence. Should there be more context for it?